charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.69.3
+version: 3.69.4
```

Comment:
NIT: is this a minor update ?

Reply:
Hopefully someone from container-ecosystems will be able to give us a definitive answer on this.